schema/workerschema/v1.json

```diff
@@ -254,6 +254,19 @@
           "response": {
             "$ref": "ClientWithSecret"
           }
+        },
+        "Revoke": {
+          "id": "dex.Client.Revoke",
+          "description": "Revoke all refresh tokens issues to the client for the authenticated user.",
+          "httpMethod": "DELETE",
+          "path": "client/refresh/{clientid}/revoke",
```

Comment:
please do remove it - you can still use the path for other things, just not DELETE.

Also, I was asking for the opposite, and some consistency. This refresh token is something that belongs to the user really, not the client. 

so something like

DELETE /account/${user_id}/refresh/${client_id}   for revoking

and 

GET /account/${user_id}/refresh/${client_id}  for listing.